lib/src/util/yust_file_handler.dart

```diff
@@ -375,6 +375,7 @@ class YustFileHandler {
     var fileData = _getFileData(cachedFile.name!, attribute);
 
     fileData['name'] = cachedFile.name;
+    fileData['modifiedAt'] = cachedFile.modifiedAt;
```

Comment:
When a file is loaded from disk (because the app was restarted) cachedFile.modifiedAt is empty. (The date is not saved on disk.)
A solution would be: Reassign the time in line 378 with: `Yust.helpers.utcNow()`